routers/router.lon1.yml

```
@@ -9,7 +9,7 @@
   wireguard:
     remote_address: dn10.uk.surgebytes.com
     remote_port: 30207
-    public_key: 4I0Lwgj9/dp6e9bIIdJDpkTN2HoAEz5+/pkl8zTd01g=
+    public_key: cHGD2aC5xkDW5n6L3EX35tqWaRPAktTQqbFETBNueS8= 
```

```suggestion
    public_key: cHGD2aC5xkDW5n6L3EX35tqWaRPAktTQqbFETBNueS8=
```

Remove trailing space after public key.